docs/_book.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+upper_tabs:
+  # Tabs left of dropdown menu
+  - include: /_upper_tabs_left.yaml
+  - include: /api_docs/_upper_tabs_api.yaml
+  # Dropdown menu
+  - name: Resources
+    path: /resources
+    is_default: true
+    menu:
+      - include: /resources/_menu_toc.yaml
+    lower_tabs:
+      # Subsite tabs
+      other:
+        - name: tutorials
+          contents:
+            - title: Triplet loss
+              path: /addons/tutorials/losses_triplet
+            - title: Image Ops
+              path: /addons/tutorials/image_ops
```

Comment:
@lamberta So this part I'm still unclear on. Where does the directory "addons" come from? Is it just the repository name?

Does it need to be `addons/docs/tutorials/*`?

Reply:
"/addons/tutorials/losses_triplet" is correct.

The workflow is "take a directory and put it on the site".

So we'll set up the tools to take "https://github.com/tensorflow/addons/tree/master/docs" and put it on the site as "tensorflow.org/addons".

So two things to remember about links for that:

1. Use relative links when linking between files within the directory (so users keep their github/colab/tensorflow.org context). Use full URLs if you need to link to something outside of the directory. Never use absolute path links.

2.  In notebooks and markdown `tfa.thing` will be auto-linked to the appropriate api-page, if it exists.

Reply:
Great thanks. So just to be clear will the utilized directory always come from `master` branch? At the moment we only generate docs for our release branches, but that'd probably require you pulling down a new directory for each release.

Reply:
@MarkDaoust @lamberta Friendly bump on this question and then we can look to merge this. Are we required to have the latest releases API docs on our master branch?

Reply:
> Are we required to have the latest releases API docs on our master branch?

No, we've simplified the setup. There is now no requirement to have api_docs in you repo at all (now). When we generate the site, we'll run your `build_docs.py` script against your latest pip package, to generate the `api_docs` pages.   

>So just to be clear will the utilized directory always come from master branch?

Yes. While api reference material is generated from your pip package, we pull the rest of the docs from master so that doc-fixes don't need to wait for a release.

The one caution there is that notebooks demonstrating features for a future release need to be sure to `pip install` an appropriate nightly or preview package, but if you're test running things in colab, for example, that should be easy to remember.